### DISCUSSION TITLE
DuckDB limited support?

### DISCUSSION THREAD

**Discussion by thadguidry** [9tigerio/db2rest#212] [Ideas]
[DuckDB](https://duckdb.org/docs/api/java) is a fast analytical OLAP database and used heavily in research and the scientific community where the needs for sharing via REST interfaces has become more important.  I'm not 100% sure if the Arrow Vector streaming where DuckDB also offers Arrow streaming with batches, would be useful or not, so we might initially skip supporting that.  However, the [Appender class](https://duckdb.org/docs/data/overview#appender-c-and-java) is very useful for bulk import and completely avoids needing SQL.

**Comment by kdhrubo** [comment]
DuckDB can be supported easily in full glory because it has JDBC driver. All features are available with this JDBC type 4 driver
